res/job_queue/job_queue_node.py

```diff
@@ -178,7 +174,6 @@ def _job_monitor(self, driver, pool_sema, max_submit):
             ):
                 self._start_time = time.time()
             time.sleep(1)
-            self.update_status(driver)
             if self._should_be_killed():
                 self._kill(driver)
```

Comment:
it seems to me that the call to `update_status` in 181 does not need to replaced by anything, as `_should_be_killed` inspect `thread_status` (and `runtime`) and does not depend on `status`. I might be wrong though...

Reply:
I would suggest to use the refactored call here

https://github.com/equinor/ert/blob/175a265d68020b650eb0d46fd64c6929a10f8217/res/job_queue/job_queue_node.py#L119

and drop the internal method `_get_status()` together with its binding to C++.